File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        Set<Integer> set = new LinkedHashSet<>();
        for(int i=0 ; i < nums.length ; i++){
            set.add(nums[i]);
        }

        int ind = 0;

        for(int i : set){
            nums[ind++] = i;
        }

        return ind;
        
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        uni = set()
        for i in nums:
            uni.add(i)
        uni = sorted(list(uni))
        ind=0
        for i in uni:
            nums[ind] = i
            ind+=1
        return ind